js/components/ReportSummaryForm.vue

```diff
@@ -17,13 +17,13 @@
       <div class="row form-group">
         <div class="col">
           <label for="title">Title</label>
-          <input id="title" name="title" v-model="formTitle" type="text" class="form-control">
+          <input id="title" name="title" v-model.trim="model.title" type="text" class="form-control">
```

Comment:
This is really neat that you can trim it right here.

Reply:
It's definitely handy. It's worth reviewing the `v-model` modifiers in the documentation occasionally:

https://vuejs.org/v2/guide/forms.html#Modifiers

Same for the event modifiers:

https://vuejs.org/v2/guide/events.html#Event-Modifiers